### ISSUE TITLE
Ability for Admin to Assume User's Login

### ISSUE THREAD

**Issue by jamesn81** [esotalk/esoTalk#397]
It'd be great if Admin users could "assume" the user's login to change their settings/info.

For example, I have some users who are not super computer savvy, and who need to subscribe to certain things or change their own settings, but who can't.  As the Admin, I could quickly solve issues for them, but unless I know their password ... I can't action a lot of things.

Not sure if this is a core thing or a plugin thing, but I would've thought the former.

**Comment by tvb** [comment]
Hmm I think this feature is going beyond the purpose of a forum. Other than that it could be possible by a plugin I suppose.

**Comment by jamesn81** [comment]
Yeah it's possible it might go out of the scope for esoTalk/Flarum, but it's a default inclusion with phpBB and it helps quite a bit in terms of managing users who aren't experienced forum users.

So on phpBB when you click on a user there is an option "Test out user's permissions" to make sure that when you create someone that they are only seeing what they are meant to see, or so that you can help them with settings etc.
